### ISSUE TITLE
Dependency Builds

### ISSUE THREAD

**Issue by franleplant** [VundleVim/Vundle.vim#739]
Hi! Thanks a lot for your work on Vundle!

Is currently any way of automatically running build scripts after installing dependencies? [example](https://github.com/Shougo/dein.vim)

A very common case is You Complete Me where after cloning the repo you need to run `./install.py --xx-complete`, is there anyway of teaching Vundle to do that?

Thanks a lot!
Fran

**Comment by nfischer** [comment]
I don't believe vundle has a way of doing this, currently. There's probably a way to write your own vim command, however, which runs `:PluginInstall!` followed by `:!./install.py --xx-complete`. Or you can look to other plugin managers, some of which offer this feature.
